README.md

```diff
@@ -1,9 +1,14 @@
-# [Nextcloud](https://nextcloud.com) Android Library [![Build Status](https://drone.nextcloud.com/api/badges/nextcloud/android-library/status.svg)](https://drone.nextcloud.com/nextcloud/android-library) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d9f94f04e0f447a6b21c0ae08f6f7594)](https://www.codacy.com/app/Nextcloud/android-library?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=nextcloud/android-library&amp;utm_campaign=Badge_Grade)
+# [Nextcloud](https://nextcloud.com) Android Library v2 [![Build Status](https://drone.nextcloud.com/api/badges/nextcloud/android-library/status.svg)](https://drone.nextcloud.com/nextcloud/android-library) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d9f94f04e0f447a6b21c0ae08f6f7594)](https://www.codacy.com/app/Nextcloud/android-library?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=nextcloud/android-library&amp;utm_campaign=Badge_Grade)
 
 ## Introduction
 Using Nextcloud Android library it will be the easiest way to communicate with Nextcloud servers.
 Add this library in your project and integrate your application with Nextcloud seamlessly.
 
+## Android Library v2
+Starting from 01.10.2019 we will not actively develop our old library (v1), but maintain it until 01.10.2021 with bug fixes.

```

Comment:
Any reason for supporting v1 for another 2 years? Used by customers?

Reply:
No, they use what we give them ;-)
This is just a nice service for other apps using our lib. 

As this is only about bug fixes and we do not backport any new features, this should be rather little work.